.github/workflows/ci_tests.yaml

```diff
@@ -96,8 +96,8 @@ jobs:
           mamba install gmt=6.2.0 numpy=${{ matrix.numpy-version }} \
                         pandas xarray netCDF4 packaging \
                         ${{ matrix.optional-packages }} \
-                        coverage[toml] dvc=2.3.0 make pytest>=6.0 \
-                        pytest-cov pytest-mpl sphinx-gallery
+                        coverage dvc=2.3.0 make pytest>=6.0 \
```

Comment:
Do you know why we have `coverage` here, but not in "ci_tests_dev.yaml"?

Reply:
Well, `git blame` shows that `coverage[toml]` was added by you in #1033, so I'll need to ask you that! We could probably remove`coverage` though, since [`pytest-cov`](https://anaconda.org/conda-forge/pytest-cov/files?version=2.12.1) depends on it.

Reply:
😄 Let's remove it and see everything works well.